docs/developers/documentation/index.md

```diff
@@ -0,0 +1,209 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+theme:
+  intro: This guide will walk you through submitting a new document to napari.org.
+---
+
+# Contributing Documentation
+
+This guide will teach you how to submit new documents to napari's usage documentation.
+
+## Prerequisites
+
+- Familiarity with `git`
+- A [GitHub](https://github.com) account
+## 0. Before you start
+
+If you'd like to contribute a brand new document to our usage section, it might be worth [opening an issue](https://github.com/napari/napari/issues/new?assignees=&labels=documentation&template=documentation.md&title=)
+ on our repository first to discuss the content you'd like to see and get some early feedback from the community.
+The napari team can also suggest what type of document would be best suited, and whether there are already
+existing documents that could be expanded to include the content you think is lacking. 
+
+Examples of documents you might want to contribute are:
+
+- **Explanations** (in `napari/docs/guides`): in depth content about napari architecture, development choices and some complex features 
+- **Tutorials** (in `napari/docs/tutorials`): detailed, reproducible step by step guides, usually combining multiple napari features to complete a potentially complex task
+- **How-tos** (in `napari/docs/howtos/`): simple step by step guides demonstrating the use of common features
+- **Getting started** (in `napari/docs/tutorials/fundamentals`): these documents are a mix of tutorials and how-tos covering the fundamentals of installing and working with napari for beginners
+
+```{admonition} Got materials for a workshop?
+:class: tip
+
+If you already have teaching materials e.g. recordings, slide decks or jupyter notebooks
+hosted somewhere, you can add links to these on our [napari workshops](../../further-resources/napari-workshops.md) page.
+```
+
+If you are writing a document whose content is mostly text,
+you can write a plain markdown document and skip straight to [Step #4 - Update TOC](#4-update-toc).
+If you are writing a how-to guide or tutorial that requires executing code or working with the napari viewer, follow
+the steps below to prepare your document.
+
+### Prerequisites for contributing documentation with code
+- [Jupyter notebook](https://jupyter.org/) installed
+- Familiarity with Jupyter notebooks (code cells and markdown cells)
+- Familiarity with using napari through a Jupyter notebook
```

Comment:
I think jupytext should be added to the requirements here, not below. Jupytext can be used to open the md files directly, without need to set up pairing.

Reply:
Yeah so my issue is that the workflow with pairing is too complex. I'm not sure if you know that with jupytext installed you can just *open* the markdown file directly as if it was an ipynb! Pairing is useful when you have *some* use for the ipynb itself, such as saving the outputs. But when you are going to contribute to napari.org, you *only* want the markdown. So imho the recommended workflow is:

- if you already have a notebook:
  - run `jupytex your-notebook.ipynb --to myst`
  - *delete* your-notebook.ipynb
  - launch jupyter notebook, open the .md, and edit as needed
  - commit, push, create PR
- If you are starting a notebook from scratch:
  - copy the template.md to whatever file you want to create
  - launch jupyter notebook, open the .md, and edit as needed
  - commit, push, create PR

What do you think?

Reply:
Yes perfect. Will update first thing in the morning!